app/features-json/setup_json_controller.rb

```diff
@@ -8,5 +8,109 @@ class SetupJSONController < APIController
     get "#{HOME}/configured", authenticate: false do
       return json(Services.onboarding_service.correct_setup?)
     end
+
+    # TODO: we need to protect this API endpoint
+    # one the onboarding user flow, we don't have any accounts yet
+    # however, we need to prohibit sending a POST request to this API endpoint
+    # after CI was already set up
+    post HOME.to_s, authenticate: false do
+      # we set this as default so that we don't need duplicate code for error detection
+      params["bot_account"] ||= {}
```

Comment:
nit: maybe a new var so later on we don't have to do `params[..]`?

Reply:
I only default to `{}` to easily access nested hashes later in the code. We can also update the code base to use `.fetch`